### PR TITLE
Tailwind case study/tailwind case study.md

### PR DESCRIPTION
**Pull request by Bavy89**
tailwind case study - Bave Ado

### CODE REVIEW

**Comment by Bavy89** [comment]
tailwind case study - Bave Ado

**Review by sulenchy — CHANGES_REQUESTED** [review]
Good presentation. Straight to the point. However, it is not a good one for a beginner. It is lacking in the aspect of comprehension. A good write-up on a CSS framework should should some code couple with some image.
So, I recommend you demonstrate styling a component.

@Bavy89

**Review by sulenchy — COMMENTED** [review]
Also, I think this content should be kept in framework directory. You can create `css` sub-directory under `frameworks`. Like `/framworks/css/tailwind-case-study.md`

**Comment by Bavy89** [comment]
hi i have added new PR
https://github.com/Bavy89/development-platforms/blob/tailwind-case-study/tailwind-case-study.md/src/content/docs/tailwind-case-study/tailwind-case-study.md

File: src/content/docs/tailwind-case-study/tailwind-case-study.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
+---
+title: Tailwind css
+author: Bavy Ado <Bavy89>
+tags: tailwind-css, frontend, css-framework, case-study
+---
+
```
Comment (sulenchy) [inline src/content/docs/tailwind-case-study/tailwind-case-study.md:6]: @Bavy89 Good job for implementing the feedback. However, now you have a duplicate. You need to remove the old one. The new one `[src/content/docs/frameworks/css/tailwind-case-study.md](https://github.com/NoroffFEU/development-platforms/pull/291/files#diff-caf64c6ec57e6a2da2e16c363b29a405b78d43a312deb01a1a482518e91ab174)` looks good to me.

**Review by sulenchy — APPROVED** [review]

**Pull request merged**